### ISSUE TITLE
sharing example code

### ISSUE THREAD

**Issue by zirho** [kesha-antonov/react-native-zoom-reanimated#6]
Thanks for the awesome project.

Could you share the code that you shown in the video?

It's okay if you could not. 
Then, can you tell me what you used for sliding pages?

Thanks in advance!

**Comment by kesha-antonov** [comment]
> Thanks for the awesome project.
> 
> 
> 
> Could you share the code that you shown in the video?
> 
> 
> 
> It's okay if you could not. 
> 
> Then, can you tell me what you used for sliding pages?
> 
> 
> 
> Thanks in advance!

You mean with list of images?

**Comment by zirho** [comment]
Yes, you showcased the swiping between images. I wonder what you used to implement that. 
Thanks!

**Comment by kesha-antonov** [comment]
Sure, I'll do that. It's just simple FlatList.

**Comment by ammarfaris** [comment]
yeah, can you share the example code please @kesha-antonov

**Comment by virajpsimformsolutions** [comment]
can you share the example code please with flatlist? @kesha-antonov
